gateway/1.6/modules/ROOT/pages/policies-reorder.adoc

```diff
@@ -1,18 +1,60 @@
-= Reordering Policies
+= Ordering Policies
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :page-aliases: api-manager::re-order-policies-task.adoc, policies-included-reorder.adoc, policies::policies-included-reorder.adoc, policies::policies-reorder.adoc
 
-To execute certain policies before others, you can configure the order of execution of included and custom API-level policies. However, you cannot configure API-level policies to execute before automated policies.
+To execute certain policies before others, you can configure the order of execution of included and custom policies. 
 
-For Flex Gateway running in Connected Mode, you reorder policies via API Manager. For Flex Gateway running in Local Mode you order policies via a `PolicyBinding` YAML resource.
+You can configure both included and custom policies to execute as automated policies or API-level policies. Automated policies are policies that are applied to all API instances deployed on the Flex Replica and execute before API-level policies. API-level policies are policies that are only applied to a specific API instance.
+
+For Flex Gateway running in Connected Mode, you order policies via API Manager. For Flex Gateway running in Local Mode you order policies via a `PolicyBinding` YAML resource.
 
 IMPORTANT: You cannot configure the execution order of the xref:policies-included-cors.adoc[Cross-Origin Resource Sharing (CORS) policy]. The CORS policy executes before automated and API-level policies.
 
-== Flex Gateway in Connected Mode
+== Policy Execution Order
+
+Policies can either execute on the API request, the API response, or on both. Policies execute in order during the API request and in inverse order during the API response. If a policy does not execute in a certain direction, Flex Gateway skips the policy in that direction.
+
+For example, if you have four policies ordered:
+
+. Automated Policy 1
+. Automated Policy 2
+. API-Level Policy 1
+. API-Level Policy 2
+
+The policies execute in the following order on the response:
+
+. API-Level Policy 2
+. API-Level Policy 1
+. Automated Policy 2
+. Automated Policy 1
+
+[[automated-policies]]
+== Order Automated Policies
+
+Refer to the following sections for your relevant mode.
```

Comment:
Suggest either adding more information here or removing this level and elevating the H3s to H2s.

Reply:
Or maybe, since Connected mode isn't really much, just remove the H3s.

The idea is to flatten the hierarchy (elevate headings to appear in the right nav) and remove empty/fluffy sections.